packages/appium/test/parser-specs.js

```diff
@@ -20,11 +20,11 @@ describe('parser', function () {
     });
 
     it('should accept only server and driver subcommands', function () {
-      p.parse_args([]);
-      p.parse_args(['server']);
-      p.parse_args(['driver', 'list']);
-      (() => p.parse_args(['foo'])).should.throw();
-      (() => p.parse_args(['foo --bar'])).should.throw();
+      p.parseArgs([]);
```

Comment:
`parseArgs()` is the canonical name of this method.  `parse_args()` is deprecated (and should probably just be removed). vestige from calling into `argparse` directly